civis/KYC/kyc_permits_to_esri_v2.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+#--Pulling Last 6 Months of Commercial & Apartment Data
+
+
+import os
+pwd=os.getcwd()
+import sys
+#!{sys.executable} -m pip install sodapy
+from sodapy import Socrata
+import pandas as pd
+import numpy as np
+from arcgis.gis import GIS
+from shapely.geometry import Point
+import geopandas as gpd
+from arcgis.features import FeatureLayerCollection
+import intake_civis
+import datetime
+
+lahub_user = os.environ["LAHUB_ACC_USERNAME"]
+lahub_pass = os.environ["LAHUB_ACC_PASSWORD"]
+socrata_token = 'LJ60SFL7ZqoC4IWosLhEmJV2a'
+socrata_user = os.environ["SOCRATA_ACC_USERNAME"]
+socrata_pass = os.environ["SOCRATA_ACC_PASSWORD"]
+permit_layer = '48fca217dd5a410bbfd6ce0abcdd3a26'
+pwd = os.getcwd()
+OUTPUT_FILE = pwd + "/Building and Safety Permits, Last 6 Months.csv"
+
+def prep_permit_data(file,token,user,pas):
+    client = Socrata("data.lacity.org", token, username=user, password=pas)
+    df = pd.DataFrame(client.get('n9nq-vewq', limit=10000000))
```

Comment:
Are we guaranteed that by limiting to ten million rows, we get all requests from the last six months? If we do get back ten million rows, might it crash the instance we are running?

It may be worth executing a SoQL query to get a more targeted subset, rather than reading them all into memory. Unless you already tried that?